src/components/Book.js

```diff
@@ -55,19 +55,24 @@ const Wrapper = styled.li`
   display: flex;
   justify-content: space-between;
   gap: 0.5rem;
-  min-height: 180px;
+  height: 180px;
+  padding-right: 5px;
   box-shadow: 5px 4px 10px 0px rgba(0, 0, 0, 0.25);
+  -webkit-box-shadow: 5px 4px 10px 0px rgba(0, 0, 0, 0.25);
```

Comment:
As you're reusing this style in some of your components, you might want to consider making use of css variables:

In your GlobalStyle component you can set custom variables like so:
```
const GlobalStyles = createGlobalStyle`
    :root {
        --box-shadow: 5px 4px 10px 0px rgba(0, 0, 0, 0.25);
    }
```

and then place them in your components like this:
```
box-shadow: var(--box-shadow);
```

That way you don't have to adjust every single styled component manually when updating the (for example) box-shadow-property.

Reply:
You're right! Life could be so much easier with using css variables! Changed it for box shadow. Will also think about switch to css variables for the colors. Thank you! ❤️